yt_dlp/extractor/medaltv.py

```diff
@@ -44,15 +45,14 @@ class MedalTVIE(InfoExtractor):
             'timestamp': 1603165266,
             'upload_date': '20201020',
             'uploader_id': '10619174',
-            'thumbnail': 'https://cdn.medal.tv/10619174/thumbnail-34934644-720p.jpg?t=1080p&c=202042&missing',
```

Comment:
Add the thumbnail line back, even if it makes the test fail. Thumbnail extraction is failing and will need to be fixed (in a separate PR -- not your responsibility)